packages/fx-core/src/component/generator/generatorAction.ts

```diff
@@ -133,13 +133,6 @@ export const fetchTemplateZipFromLocalAction: GeneratorAction = {
 
     const data: Buffer = await fs.readFile(zipPath);
     context.zip = new AdmZip(data);
-  },
-};
-
-export const unzipAction: GeneratorAction = {
-  name: GeneratorActionName.Unzip,
-  run: async (context: GeneratorContext) => {
-    context.logProvider.debug(`Unzipping: ${JSON.stringify(context)}`);
     if (!context.zip) {
```

Comment:
This condition is meaningless. Let's remove it.

Reply:
All right.